README.md

```diff
@@ -32,12 +32,46 @@ sudo ln -s /usr/lib/llvm-15/bin/clang{,++,-format,-tidy} /usr/local/bin
 
 Python 3 is required to automatically build and link unit tests. Python 3 comes preinstalled on most Linux distributions, and is available as a package on all others.
 
-# Getting started (using Clang)
+Although not essential, to aid performance, we recommend running the OS in the non GUI mode. This can be achieved by running:
+```
+sudo raspi-config
+```
+Selecting 1 System Options, S5 Boot / Auto Login, B2 Console Autologin. Reboot and enjoy the resource saving.
```

Comment:
You could make this visually clearer by doing a numbered list of steps, or using > to separate the steps.

Reply:
added some >>